src/Popover.js

```diff
@@ -34,34 +30,66 @@ const Popover = ({
     dataTest,
     elevation,
     maxWidth,
+    observePopperResize,
+    observeReferenceResize,
     placement,
-    onBackdropClick,
-}) =>
-    createPortal(
-        <Backdrop onClick={onBackdropClick} transparent>
-            <Popper
-                dataTest={`${dataTest}-popper`}
-                placement={placement}
-                reference={reference}
-                modifiers={arrow ? arroModifiers : []}
+    onClickOutside,
+}) => {
+    const referenceElement =
+        reference instanceof Element
+            ? reference
+            : reference && reference.current
+    const [popperElement, setPopperElement] = useState(null)
+    const [arrowElement, setArrowElement] = useState(null)
+    const modifiers = useMemo(
+        () =>
+            combineModifiers(arrow, arrowElement, {
+                observePopperResize,
+                observeReferenceResize,
+            }),
+        [arrow, arrowElement]
```

Comment:
This is missing `observePopperResize` and `observeReferenceResize` as dependencies, is that intentional?  I think this should also fail the `eslint` `react-hooks/exhaustive-deps` check, is that not running here for some reason?

Reply:
It was simply an oversight.